openmdao/core/total_jac.py

```diff
@@ -335,6 +350,11 @@ def _create_in_idx_map(self, has_lin_constraints):
                 if matmat or parallel_deriv_color:
                     rhsname = name
 
+                    if parallel_deriv_color:
```

Comment:
maybe 'if parallel_deriv_color and self.debug_print' instead, since we don't need this dict if we're not in debug mode.

Reply:
Fixed.